Gopher-Industries/Team-TaskforceX
# 103-Start-Screen

Summary of Changes:

-Created new splash screen image.
-Changed fonts and weights.
-Changed margins to align with wireframe.
-Added additional comments.
-Cleaned up some code
-Changed colours to match wireframe.
-Updated buttons.
-Confirmed buttons function as required.
-Added alert function for terms of service.

Testing instructions:

-Confirm background image is visible.
-Confirm 'Get Started" button takes you to "GettingStartedInfo1".
-Confirm "I already have an account" button takes you to "login".
-Confirm clicking on "terms of service" button presents alert "No terms of service yet exists".

![landing page](https://user-images.githubusercontent.com/118961659/207197377-4f066fc2-8f14-4f24-8e76-2c773abbd02c.PNG)
![landing page 2](https://user-images.githubusercontent.com/118961659/207239886-0138542a-8139-4987-b8f9-271712b94e40.PNG)
